### ISSUE TITLE
Nested heading issues

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1900]
## Problem

There are a few situations where the programmatic render of content is causing improper heading combinations.

## Proposed Solution

Fix by creating specific templates or setting heading level variable that is passed to template.

- [x] Articles (articles, featured content paragraphs = h3, view page  = h2)
- [x] People (people, featured content paragraphs = h3, view page  = h2)
- [x] Basic Page (featured content paragraphs = h3)
- [x] Pagination should be switched to a div instead of a heading (classy view template) as we don't always know the heading content before.

**Comment by joewhitsitt** [comment]
Seeing as the basic page teaser template is not using the card component, I have created https://github.com/uiowa/uiowa/issues/1955. Basic page is already an h3 where it is being called currently and is not an issue.
